ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -1294,7 +1579,6 @@
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = ReactiveCocoaTests/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
```

Comment:
I am not so sure about `LD_RUNPATH_SEARCH_PATHS`. But in many cases the additions in the target are the same as the inherited paths.